Securitytxt/Model/Config/Backend/Validate.php

```diff
@@ -123,4 +147,85 @@ private function validateUrlField(string $fieldName, string $fieldValue): void
             );
         }
     }
+
+    /**
+     * Get Value from form or inheriting value.
+     *
+     * @param array $fieldData
+     * @return string
+     */
+    private function getDataValue(array $fieldData): string
+    {
+        $result = '';
+        if (isset($fieldData['value'])) {
+            $result = $fieldData['value'];
+        }
+
+        return $result;
```

Comment:
```suggestion
      return $fieldData['value'] ?: '';
```

Reply:
Seems this construction not working,
if not exist array element with key  - value

Updated with following construction
-
`return isset($fieldData['value']) ? $fieldData['value'] : '';`